testing/trino-product-tests/src/main/java/io/trino/tests/product/deltalake/TestDeltaLakeDatabricksCreateTableCompatibility.java

```diff
@@ -312,6 +313,20 @@ public void testCreateTableWithColumnCommentOnDelta()
         }
     }
 
+    @Test(groups = {DELTA_LAKE_DATABRICKS, PROFILE_SPECIFIC_TESTS})
+    @Flaky(issue = DATABRICKS_COMMUNICATION_FAILURE_ISSUE, match = DATABRICKS_COMMUNICATION_FAILURE_MATCH)
+    public void testCreateTableWithAnonymousRowType()
+    {
+        String tableName = "test_create_anonymous_row_type_" + randomNameSuffix();
+        String tableDirectory = "databricks-compatibility-test-" + tableName;
+
+        assertThatThrownBy(() -> onTrino().executeQuery(format("CREATE TABLE delta.default.%s AS SELECT ROW(123) AS c1 with (location = 's3://%s/%s')",
```

Comment:
Please use `onDelta` instead of `onTrino`. Creating a table in Trino is already covered in TestDeltaLakeConnectorTest.java.

Reply:
I'm not very familiar with Delta Lake, so I'm having a bit of difficulty understanding the purpose of this test. What kind of compatibility should the code be testing?